### ISSUE TITLE
default app.yaml file doesn't work.

### ISSUE THREAD

**Issue by kirilligum** [GoogleCloudPlatform/appengine-flask-skeleton#7]
tl;dr : 

when I uncomment the following lines in app.yml in the github clone, everything works:
```
#libraries:
#- name: jinja2
#  version: latest
```

Hi,

I have fedora 20 (just upgraded from 19) running in a virtualbox under windows 8.

I'm following gae tutorial https://developers.google.com/appengine/docs/python/getting-started-with-flask

there are two ways to run the hello world. 1) git and install libraries and 2) get the zip.
method 2 works but method 1 doesn't.

since I'm using fedora 20, I have pip 1.4 and python 2.7

    17:26:39 ~/gae 
    > git clone --recursive https://github.com/GoogleCloudPlatform/appengine-python-flask-skeleton
    Cloning into 'appengine-python-flask-skeleton'...
    remote: Reusing existing pack: 376, done.
    remote: Total 376 (delta 0), reused 0 (delta 0)
    Receiving objects: 100% (376/376), 779.61 KiB | 403.00 KiB/s, done.
    Resolving deltas: 100% (38/38), done.
    Checking connectivity... done.
    17:27:03 ~/gae 
    > cd appengine-python-flask-skeleton/
    17:27:31 ~/gae/appengine-python-flask-skeleton 
    > pip install -r requirements.txt -t lib
    Downloading/unpacking Flask==0.10 (from -r requirements.txt (line 7))
      Downloading Flask-0.10.tar.gz (544kB): 544kB downloaded
      Running setup.py egg_info for package Flask
        
        warning: no files found matching '*' under directory 'tests'
        warning: no previously-included files matching '*.pyc' found under directory 'docs'
        warning: no previously-included files matching '*.pyo' found under directory 'docs'
        warning: no previously-included files matching '*.pyc' found under directory 'tests'
        warning: no previously-included files matching '*.pyo' found under directory 'tests'
        warning: no previously-included files matching '*.pyc' found under directory 'examples'
        warning: no previously-included files matching '*.pyo' found under directory 'examples'
        no previously-included directories found matching 'docs/_build'
        no previously-included directories found matching 'docs/_themes/.git'
    Downloading/unpacking Werkzeug>=0.7 (from Flask==0.10->-r requirements.txt (line 7))
      Downloading Werkzeug-0.9.4.tar.gz (1.1MB): 1.1MB downloaded
      Running setup.py egg_info for package Werkzeug
        
        warning: no files found matching '*' under directory 'werkzeug/debug/templates'
        warning: no files found matching '*' under directory 'tests'
        warning: no previously-included files matching '*.pyc' found under directory 'docs'
        warning: no previously-included files matching '*.pyo' found under directory 'docs'
        warning: no previously-included files matching '*.pyc' found under directory 'tests'
        warning: no previously-included files matching '*.pyo' found under directory 'tests'
        warning: no previously-included files matching '*.pyc' found under directory 'examples'
        warning: no previously-included files matching '*.pyo' found under directory 'examples'
        no previously-included directories found matching 'docs/_build'
    Downloading/unpacking Jinja2>=2.4 (from Flask==0.10->-r requirements.txt (line 7))
      Downloading Jinja2-2.7.2.tar.gz (378kB): 378kB downloaded
      Running setup.py egg_info for package Jinja2
        
        warning: no files found matching '*' under directory 'custom_fixers'
        warning: no previously-included files matching '*' found under directory 'docs/_build'
        warning: no previously-included files matching '*.pyc' found under directory 'jinja2'
        warning: no previously-included files matching '*.pyc' found under directory 'docs'
        warning: no previously-included files matching '*.pyo' found under directory 'jinja2'
        warning: no previously-included files matching '*.pyo' found under directory 'docs'
    Downloading/unpacking itsdangerous>=0.21 (from Flask==0.10->-r requirements.txt (line 7))
      Downloading itsdangerous-0.24.tar.gz (46kB): 46kB downloaded
      Running setup.py egg_info for package itsdangerous
        
        warning: no previously-included files matching '*' found under directory 'docs/_build'
    Downloading/unpacking markupsafe (from Jinja2>=2.4->Flask==0.10->-r requirements.txt (line 7))
      Downloading MarkupSafe-0.19.tar.gz
      Running setup.py egg_info for package markupsafe
        
    Installing collected packages: Flask, Werkzeug, Jinja2, itsdangerous, markupsafe
      Running setup.py install for Flask
        
        warning: no files found matching '*' under directory 'tests'
        warning: no previously-included files matching '*.pyc' found under directory 'docs'
        warning: no previously-included files matching '*.pyo' found under directory 'docs'
        warning: no previously-included files matching '*.pyc' found under directory 'tests'
        warning: no previously-included files matching '*.pyo' found under directory 'tests'
        warning: no previously-included files matching '*.pyc' found under directory 'examples'
        warning: no previously-included files matching '*.pyo' found under directory 'examples'
        no previously-included directories found matching 'docs/_build'
        no previously-included directories found matching 'docs/_themes/.git'
      Running setup.py install for Werkzeug
        
        warning: no files found matching '*' under directory 'werkzeug/debug/templates'
        warning: no files found matching '*' under directory 'tests'
        warning: no previously-included files matching '*.pyc' found under directory 'docs'
        warning: no previously-included files matching '*.pyo' found under directory 'docs'
        warning: no previously-included files matching '*.pyc' found under directory 'tests'
        warning: no previously-included files matching '*.pyo' found under directory 'tests'
        warning: no previously-included files matching '*.pyc' found under directory 'examples'
        warning: no previously-included files matching '*.pyo' found under directory 'examples'
        no previously-included directories found matching 'docs/_build'
      Running setup.py install for Jinja2
        
        warning: no files found matching '*' under directory 'custom_fixers'
        warning: no previously-included files matching '*' found under directory 'docs/_build'
        warning: no previously-included files matching '*.pyc' found under directory 'jinja2'
        warning: no previously-included files matching '*.pyc' found under directory 'docs'
        warning: no previously-included files matching '*.pyo' found under directory 'jinja2'
        warning: no previously-included files matching '*.pyo' found under directory 'docs'
      Running setup.py install for itsdangerous
        
        warning: no previously-included files matching '*' found under directory 'docs/_build'
      Running setup.py install for markupsafe
        
        building 'markupsafe._speedups' extension
        gcc -pthread -fno-strict-aliasing -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/usr/include/python2.7 -c markupsafe/_speedups.c -o build/temp.linux-x86_64-2.7/markupsafe/_speedups.o
        gcc -pthread -shared -Wl,-z,relro build/temp.linux-x86_64-2.7/markupsafe/_speedups.o -L/usr/lib64 -lpython2.7 -o build/lib.linux-x86_64-2.7/markupsafe/_speedups.so
    Successfully installed Flask Werkzeug Jinja2 itsdangerous markupsafe
    Cleaning up...
    17:27:42 ~/gae/appengine-python-flask-skeleton 
    > 

Then, I run the server:

    > dev_appserver.py .
    INFO     2014-04-16 00:28:41,067 sdk_update_checker.py:242] Checking for updates to the SDK.
    INFO     2014-04-16 00:28:41,271 sdk_update_checker.py:286] This SDK release is newer than the advertised release.
    INFO     2014-04-16 00:28:41,345 api_server.py:171] Starting API server at: http://localhost:45466
    INFO     2014-04-16 00:28:41,362 dispatcher.py:182] Starting module "default" running at: http://localhost:8080
    INFO     2014-04-16 00:28:41,363 admin_server.py:117] Starting admin server at: http://localhost:8000

Then, I go to `http://localhost:8080/` and I get:

    ERROR    2014-04-16 00:29:20,777 wsgi.py:262] 
    Traceback (most recent call last):
      File "/home/kirill/gae/google_appengine/google/appengine/runtime/wsgi.py", line 239, in Handle
        handler = _config_handle.add_wsgi_middleware(self._LoadHandler())
      File "/home/kirill/gae/google_appengine/google/appengine/runtime/wsgi.py", line 298, in _LoadHandler
        handler, path, err = LoadObject(self._handler)
      File "/home/kirill/gae/google_appengine/google/appengine/runtime/wsgi.py", line 84, in LoadObject
        obj = __import__(path[0])
      File "/home/kirill/gae/appengine-python-flask-skeleton/main.py", line 4, in <module>
        from flask import Flask
      File "/home/kirill/gae/appengine-python-flask-skeleton/lib/flask/__init__.py", line 19, in <module>
        from jinja2 import Markup, escape
      File "/home/kirill/gae/appengine-python-flask-skeleton/lib/jinja2/__init__.py", line 33, in <module>
        from jinja2.environment import Environment, Template
      File "/home/kirill/gae/appengine-python-flask-skeleton/lib/jinja2/environment.py", line 13, in <module>
        from jinja2 import nodes
      File "/home/kirill/gae/appengine-python-flask-skeleton/lib/jinja2/nodes.py", line 18, in <module>
        from jinja2.utils import Markup
      File "/home/kirill/gae/appengine-python-flask-skeleton/lib/jinja2/utils.py", line 520, in <module>
        from markupsafe import Markup, escape, soft_unicode
    ImportError: No module named markupsafe
    INFO     2014-04-16 00:29:20,788 module.py:627] default: "GET / HTTP/1.1" 500 -

Then, I kill it.

here is the contents of the `lib` directories for both cases: 1) and 2):

when I uncomment the following lines in app.yml in the case 1), everything works:
```
#libraries:
#- name: jinja2
#  version: latest
```

**Comment by jonparrott** [comment]
I couldn't reproduce. Feel free to re-open if this is still happening.
